requirements.txt

```diff
@@ -1,6 +1,6 @@
-pylint==1.4.4
+-e git+git://github.com/PyCQA/pylint.git@master#egg=pylint
```

Comment:
Yes, this is a known issue because currently we don't have a release in pylint, but now I have created the question to create a release: https://github.com/PyCQA/pylint/issues/874#issuecomment-210949485